WORKSPACE

```diff
@@ -104,3 +104,64 @@ load("@com_github_jupp0r_prometheus_cpp//:repositories.bzl",
 # Load dependencies individually since we load some of them above.
 load_prometheus_client_model()
 load_civetweb()
+
+# Curl library - used by zipkin exporter.
+new_http_archive(
+    name = "com_github_curl",
+    urls = ["https://github.com/curl/curl/archive/master.zip"],
+    strip_prefix = "curl-master",
+    build_file_content =
+"""
+load("@io_opencensus_cpp//opencensus/exporters/trace/zipkin:curl.bzl", "CURL_COPTS")
+package(features = ['no_copts_tokenization'])
+
+config_setting(
+    name = "windows",
+    values = {"cpu": "x64_windows"},
+    visibility = [ "//visibility:private" ],
+)
+
+cc_library(
+    name = "curl",
+    srcs = glob([
+        "lib/*.c",
+        "lib/vauth/*.c",
```

Comment:
Should this be lib/**/*.c or are there dirs being excluded?

Reply:
Done.

Reply:
Done.